logspout/build.yml

```diff
@@ -0,0 +1,5 @@
+repository: monasca/logspout
+variants:
+  - tag: 3.2.3
```

Comment:
I'd say ```0.0.1-3.2.3``` or just ```0.0.1``` and have ```LOGSPUT_VERSION``` down in arguments of the build.
I wrote a comment above about that but I'd add here that I lean a bit toward simpler semver ```0.0.1```.

Reply:
Done